src/NServiceBus.Transport.RabbitMQ.TransportTests/RabbitMqConnectionStringParser.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
```

Comment:
Should we link to `src/targets/RabbitMqConnectionStringParser.cs` rather than copy pasting it?

Reply:
Done :+1: